specs/_template/02_user_interface.rst

```diff
@@ -0,0 +1,40 @@
+=====================
+User interface
+=====================
+
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: SPEC_REPO#EPIC
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+
+Please specify screens for all platforms in this document.
+
+Please remove all unrelated sections to keep the document easy to read and maintain.
+
+<Screen Name>
```

Comment:
This should also be numbered, right?

Reply:
thanks to @fmrsabino suggestion, the numbers of headers are put automatically when using `.. sectnum::` directive. 👏